### PR TITLE
[PIL-979][AV] - Repayments CYA Page UK Bank Account

### PR DESCRIPTION
**Pull request by aashik-varahan**

### CODE REVIEW

File: test/controllers/repayments/RepaymentsContactByTelephoneControllerSpec.scala
Comment (javed-iqbal1) [inline test/controllers/repayments/RepaymentsContactByTelephoneControllerSpec.scala:1]: remove commented code if not needed

File: test/navigation/RepaymentNavigatorSpec.scala [hunk 1]
```diff
@@ -46,20 +53,40 @@ class RepaymentNavigatorSpec extends SpecBase {
...
+
       "go to journey recovery page if they somehow manage to submit an empty form" in {
         navigator.nextPage(UkOrAbroadBankAccountPage, None, NormalMode, emptyUserAnswers) mustBe journeyRecovery
         case object UnknownPage extends Page
-        navigator.nextPage(UnknownPage, None, NormalMode, UserAnswers("id")) mustBe routes.IndexController.onPageLoad
+        navigator.nextPage(
+          UnknownPage,
+          None,
+          NormalMode,
+          UserAnswers("id")
```
Comment (javed-iqbal1) [inline test/navigation/RepaymentNavigatorSpec.scala:82]: can we route to start page of  repayments here ?

File: test/controllers/repayments/RepaymentsContactByTelephoneControllerSpec.scala
Comment (aashik-varahan) [reply test/controllers/repayments/RepaymentsContactByTelephoneControllerSpec.scala:1]: Removed.

File: test/navigation/RepaymentNavigatorSpec.scala (hunk 1 above)
Comment (aashik-varahan) [reply test/navigation/RepaymentNavigatorSpec.scala:82]: As discussed offline, I will wait for PIL-148 to be tested before making this change.

**Review by javed-iqbal1 — APPROVED** [review]

**Review by javed-iqbal1 — APPROVED** [review]

**Pull request merged**